packages/itwin/grouping-mapping-widget/src/widget/components/GroupPropertyTable.tsx

```diff
@@ -82,13 +81,6 @@ const GroupPropertyTable = ({
               return (
                 <DropdownMenu
                   menuItems={(close: () => void) => [
-                    <MenuItem
-                      key={0}
-                      onClick={() => onGroupPropertyModify(value)}
```

Comment:
Is it no longer possible to modify the property? I think you said that you were able to bring this back.